openapi.yaml

```diff
@@ -0,0 +1,645 @@
+      description: Pass JSON containing the required parameters to create a new developer record
+      tags:
+        - Developers
+      parameters:
+        - in: body
+          name: body
+          required: true
+          schema:
+            $ref: '#/definitions/DeveloperPostBody'
+      responses:
+        '200':
+          description: Developer created successfully
+          schema:
+            $ref: '#/definitions/DeveloperResult'
+        '400':
+          description: Invalid developer object
+          schema:
+            $ref: '#/definitions/Errors'
+        '500':
+          description: Internal server error
+          schema:
+            $ref: '#/definitions/Errors'
+  /developers/{developerId}:
+    parameters:
+      - $ref: '#/parameters/authorization'
+      - name: developerId
+        in: path
+        type: integer
+        required: true
+    get:
+      summary: Get developer by its Id
+      tags:
+        - Developers
+      responses:
+        '200':
+          description: Successful response
+          schema:
+            $ref: '#/definitions/DeveloperResult'
+        '404':
+          description: Developer not found
+          schema:
+            $ref: '#/definitions/Errors'
+        '500':
+          description: Internal server error
+          schema:
+            $ref: '#/definitions/Errors'
+    patch:
+      summary: Update developer by its Id
+      tags:
+      - Developers
+      parameters:
+        - name: body
+          in: body
+          schema:
+            $ref: '#/definitions/DeveloperResource'
+      responses:
+        '200':
+          description: Developer updated successfully
+          schema:
+            $ref: '#/definitions/DeveloperResult'
+        '400':
+          description: Bad request
+          schema:
+            $ref: '#/definitions/Errors'
+        '404':
+          description: Developer not found
+          schema:
+            $ref: '#/definitions/Errors'
+        '500':
+          description: Internal server error
+          schema:
+            $ref: '#/definitions/Errors'
+    delete:
+      summary: Delete a developer record
+      tags:
+        - Developers
+      responses:
+        '200':
+          description: Developer deleted successfully
+        '404':
+          description: Developer not found
+          schema:
+            $ref: '#/definitions/Errors'
+        '500':
+          description: Internal server error
+          schema:
+            $ref: '#/definitions/Errors'
+  /reviews:
+    parameters:
+    - $ref: '#/parameters/authorization'
+    get:
+      summary: List all reviews
+      description: Get a list of all reviews or filter by query parameters
+      tags:
+        - Reviews
+      parameters:
+        - name: reviewer
+          in: query
+          type: string
+          required: false
+          description: Filter by the name of the game reviewer
+        - name: score
+          in: query
```

Comment:
You're filtering by exact core matching. Maybe use a range of scores for filtering by score.

Reply:
added min/max parameters for scores